addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -67,6 +67,31 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {mail.messaging}
+         */
+        _computeMessaging() {
+            return [['link', this.env.messaging]];
+        }
+
+        /**
+         * @private
+         * @returns {string[]}
+         */
+        _computeTextInputSendShortcuts() {
+            if (!this.thread) {
+                return;
+            }
+            const isMailingList = this.thread.model === 'mail.channel' && this.thread.mass_mailing;
+            // The goal here is to target small screen sizes as there is no shortcut
+            // to send a message for a non-mailing channel with small screen size.
```

Comment:
Following other comments I just posted, I would change this comment again.

So the reason we want to use a different shortcut in mobile is because the composer is different and it actually has a "send button", so we want enter to allow new line inside content, and send will be done on clicking the button, or using the ctrl/meta shortchut, similar to other places where we had a button.

You don't need to copy/paste this phrase, but this is the idea that should be explained in this comment.